plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -147,6 +148,8 @@ public function execute(array $data = [])
 
             throw $e;
         }
+        // Remove `Status.level` filter here to allow user object with `draft` or any other status
+        Configure::delete('Status.level');
 
         return (new GetObjectAction(['table' => $this->Users]))->execute(['primaryKey' => $user->id, 'contain' => 'Roles']);
```

Comment:
Maybe it would be better to restore the status level after getting user to consider it in any later queries 
```suggestion
        $statusLevel = Configure::read('Status.level');
        Configure::delete('Status.level');

        $user = (new GetObjectAction(['table' => $this->Users]))->execute(['primaryKey' => $user->id, 'contain' => 'Roles']);
        Configure::write('Status.level', $statusLevel);
        
        return $user;
```